Cargo.lock

```diff
@@ -1959,7 +1959,7 @@ dependencies = [
 
 [[package]]
 name = "sargon"
-version = "0.6.9"
+version = "0.6.10"
```

Comment:
lets see now if my recent git tag change of Swift release script works :D previously git tags became `0.7.0` but I hope when Swift CD finishes once I merge tihs PR that the next release is indeed `0.6.10`...